From 2624ebd1e578d392b1e3b27fdb0763b54f8a63f4 Mon Sep 17 00:00:00 2001
From: Tengfei Fan <quic_tengfan@quicinc.com>
Date: Fri, 26 Apr 2024 10:20:23 +0800
Subject: [PATCH] QCLINUX: arm64: dts: qcom: sa8775p-ride: Add board-id and
 msm-id support

Add board-id and msm-id for SA8775p platform for supporting pick
correct DTB.

Change-Id: I824aaff52fe14c0e3931ede3d352ede45060c644
Signed-off-by: Tengfei Fan <quic_tengfan@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/sa8775p-ride.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sa8775p-ride.dts b/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
index 81a7eeb9cfcd..cc883302473a 100644
--- a/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
+++ b/arch/arm64/boot/dts/qcom/sa8775p-ride.dts
@@ -14,6 +14,8 @@
 / {
 	model = "Qualcomm SA8775P Ride";
 	compatible = "qcom,sa8775p-ride", "qcom,sa8775p";
+	qcom,msm-id = <533 0x10000>, <534 0x10000>, <534 0x20000>;
+	qcom,board-id = <0x19 0>, <0x25 1>;
 
 	aliases {
 		ethernet0 = &ethernet0;
-- 
2.25.1

